docs/branding/typography.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Delete this file as there is no meaningful content. Add a typography section to the logos.md file. So many different files are not necessary.